sirepo/pkcli/admin.py

```diff
@@ -64,6 +64,28 @@ def create_examples():
                     if example.models.simulation.name not in names:
                         _create_example(example)
 
+# TODO(e-carlin): more than uid (ex email)
+def delete_user(uid):
+    """Delete a user and all of their data across Sirepo and Jupyter
+
+    This will delete information based on what is configured. So configure
+    all service (jupyterhublogin, email, etc.) that may be relevant. Once
+    this script runs all records are blown away from the db's so if you
+    forget to configure something you will have to delete manually.
+    """
+    import sirepo.server
+    import sirepo.template
+    sirepo.server.init()
```

Comment:
we need to encapsulate this...

Reply:
37af3c624

Reply:
yes... i'll spend some time this week thinking about it more.